app/views/patients/_fulfillment.html.erb

```diff
@@ -9,8 +9,10 @@
           <ul>
             <li>Clinic: <%= @patient.clinic.try(:name) %></li>
             <li><%= FUND %> Pledge Amount: $<%= @patient.fund_pledge %></li>
-            <li>Pledge generated by: <%= @patient.pledge_generated_by %></li>
-            <li>Pledge generated at: <%=@patient.created_at.strftime("%m/%d/%y") %></li>
+            <li>Pledge generated by: <%= @patient.pledge_generated_by ? @patient.pledge_generated_by : 'N/A' %></li>
+            <li>Pledge generated at: <%= @patient.pledge_generated_at ? @patient.pledge_generated_at.strftime("%m/%d/%y") : 'N/A' %></li>
+            <li>Pledge sent by: <%= @patient.pledge_sent_by ? @patient.pledge_sent_by : 'N/A' %></li>
+            <li>Pledge sent at: <%= @patient.pledge_sent_at ? @patient.pledge_sent_at.strftime("%m/%d/%y") : 'N/A'%></li>
```

Comment:
We have a method we've bolted on to the `Time` class here called `display_date` that pretty much does the same thing here, keeps our formatting consistent. Can you swap out the `strftimes` in here with that?